### PR TITLE
Make view for chat screen

### PR DESCRIPTION
**Pull request by yunakkano**
# what
- provisional coding of index.html.haml and its partials (_main_chat.html.haml / _side_bar.html.haml)
1. index.html.haml is just a wrapper file which calls partials of side bar and chat main screen.
2. _main_chat.html.haml implements :
 2.1 the header where group-name, group-members and edit-button are located,
 2.2 the message list where a message, date & time and talker of the message is shown.
 2.3 the footer where message form (with text field, image file input and send button) is located.  
3. _side_bar.html.haml
 3.1 the header where user name and icons are located
 3.2 the group list where the list of groups and lates messages of each group are presented.

- provisional coding of application.scss and its partials (_chat-main.scss / _side-bar.scss / variable.scss) the relationship between application.scss, _chat-main.scss and _side-bar.scss is identical to that of index.html.haml and its partials.
1. application.scss just imports _chat-main.scss, _side-bar.scss and other files.
2. _chat-main.scss defines style of _main_chat.html.haml
3. _side-bar.scss defines style of _side_bar.html.haml

# why
The view of chat screen is the main part of the chat-space application. Making provisional view is essential to proceed with the main development efficiently and avoid reworking in the middle of development process. 

スクリーンショット 2020-04-19 23 22 16

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'messages/index'
```
Comment (miyagit) [inline config/routes.rb:2]: こちら削除しましょう。

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```
Comment (miyagit) [inline app/views/layouts/application.html.haml:4]: こちらhashロケットを直しましょう。文字列のハッシュロケットを直した方が良い理由については下記のサイトに載ってます。
https://pikawaka.com/ruby/hash

File: config/routes.rb (hunk 1 above)
Comment (yunakkano) [reply config/routes.rb]: get ...を削除しました。

File: app/views/layouts/application.html.haml (hunk 2 above)
Comment (yunakkano) [reply app/views/layouts/application.html.haml]: hashロケットをシンボル型に直しました。

**Comment by shingoteshima** [comment]
LGTMです！！

**Pull request merged**